instagram/src/components/PostContainer/PostContainer.js

```diff
@@ -1,39 +1,44 @@
 import React from 'react';
 import Comment from '../CommentSection/CommentSection'
 import './PostContainer.css'
-import likebutton from './images/likebutton.PNG'
-import commentbutton from './images/commentbutton.PNG'
+import LikeButton from './LikeButton'
 
-function PostContainer(props) {
-    if (!props.data.length) {
-        return <h1>Loading Your Post...</h1>
+class PostContainer extends React.Component {
+    constructor(props) {
```

Comment:
Why are we setting state here? In my mind there's no reason to set the state here because we already have access to the dummyData through props.

Reply:
This is likely what's causing the problem with your search functionality.